Makefile

```diff
@@ -154,6 +171,7 @@ test\:build:
 
 .PHONY: test\:ci
 test\:ci:
+	@$(MAKE) mock
```

Comment:
todo(blocking): As discussed on discord I don't like our ci to be running tests on a different state than the current HEAD commit.